### PR TITLE
fix: invalid arch infos from native runtimes

### PR DESCRIPTION
**Pull request by alkee**

### CODE REVIEW

File: NuGetImporterForUnity/Packages/NuGet Importer/Editor/NuGetNativeImportSetting.cs [hunk 1]
```diff
@@ -133,7 +133,8 @@ string pluginPath
                 platformPath = Path.GetDirectoryName(platformPath);
             }
```
Comment (kumaS-nu) [inline NuGetImporterForUnity/Packages/NuGet Importer/Editor/NuGetNativeImportSetting.cs:134]: No longer need this code due to your update. (NuGetNativeImportSetting.cs L.130-134)

**Review by kumaS-nu — APPROVED** [review]
If you remove the no longer needed code, there is no problem. This pull request is good.

**Comment by kumaS-nu** [comment]
Thank you for your contribution.
It's been a while since I've updated, so I need to update GitHub Actions to test this repository. It may take some time to merge your pull request.
I will merge as soon as the test passes.

File: NuGetImporterForUnity/Packages/NuGet Importer/Editor/NuGetNativeImportSetting.cs (hunk 1 above)
Comment (alkee) [reply NuGetImporterForUnity/Packages/NuGet Importer/Editor/NuGetNativeImportSetting.cs]: I cleaned up a bit. Thank you.

**Review by kumaS-nu — COMMENTED** [review]
LGTM

**Comment by kumaS-nu** [comment]
A GitHub change seems to have prevented the secret from being passed to the pull request from the forked repository. The change is acceptable, so I'll merge it in.

**Pull request merged**